libc/unistd_fsdir.c

```diff
@@ -213,6 +237,14 @@ TEST(unistd_fsdir, fchdir)
 		Declared but unimplemented in libphoenix
 		https://github.com/phoenix-rtos/phoenix-rtos-project/issues/280
 	*/
+
+	/* TEAR_DOWN  is not called after TEST_IGNORE, so we have to change workdir and remove file here */
+	TEST_ASSERT_EQUAL_INT(0, chdir(testWorkDir));
```

Comment:
Uch, we should use `IGNORE_TEST` from `unity_fixture.h` instead of ignoring the test in the function body. That way `TEST_SETUP` also won't be called

Reply:
Oh, that's right `IGNORE_TEST` seems to be much better.